.chloggen/update-attribute-sqlserverreceiver.yaml

```
@@ -0,0 +1,29 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: sqlserverreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: |
+  - add `host.name` resource attributes in the exported metrics. in document it says this attribute is exported but actually not.
```

```suggestion
note: add `host.name` resource attributes in the exported metrics to match documented functionality
```

I'd prefer having two separate changelogs for this PR, one for each of these changes. It's fine to have the issue point at the same PR 👍

---

updated